docs/testing.md

```diff
@@ -10,7 +10,9 @@ An empty backend stub can be created using the following ways:
 
 * by calling `.stub` on the "real" base backend's companion object, e.g. `AsyncHttpClientZioBackend.stub` or `HttpClientMonixBackend.stub`
 * by using one of the factory methods `SttpBackendStub.synchronous` or `SttpBackendStub.asynchronousFuture`, which return stubs which use the `Identity` or standard Scala's `Future` effects without streaming support
-* by explicitly specifying the effect and supported capabilities, e.g. `SttpBackendStub[Task, MonixStreams with WebSockets](TaskMonad)`
+* by explicitly specifying the effect and supported capabilities:
+  * for Monix `SttpBackendStub[Task, MonixStreams with WebSockets](TaskMonad)`
+  * for cats `SttpBackendStub[IO, WebSockets](implicitly[MonadAsyncError[IO]])`
```

Comment:
maybe we should add ZIO as well?

Reply:
The problem with "real" backends is that there's many of them for each implementation, so we won't be able to enumerate them all. I think people will get the idea that they have to call `.stub` :)